packages/orchestration/src/cosmos-api.ts

```diff
@@ -360,19 +361,26 @@ export interface LocalAccountMethods extends StakingAccountActions {
 /**
  * Options for {@link OrchestrationAccountI} `transfer` method.
  *
+ * If specifying a custom timeout, use one of `timeoutHeight`, `timeoutTimestamp`, or `timeoutRelativeSeconds`.
+ *
  * @see {@link https://github.com/cosmos/ibc/tree/master/spec/app/ics-020-fungible-token-transfer#data-structures ICS 20 Data Structures}
  */
-export interface IBCMsgTransferOptions {
-  timeoutHeight?: MsgTransfer['timeoutHeight'];
-  timeoutTimestamp?: MsgTransfer['timeoutTimestamp'];
+export type IBCMsgTransferOptions = {
   memo?: string;
   forwardOpts?: {
     /** The recipient address for the intermediate transfer. Defaults to 'pfm' unless specified */
     intermediateRecipient?: CosmosChainAddress;
     timeout?: ForwardInfo['forward']['timeout'];
     retries?: ForwardInfo['forward']['retries'];
   };
-}
+} & (
+  | { timeoutHeight?: MsgTransfer['timeoutHeight'] }
+  | { timeoutTimestamp?: MsgTransfer['timeoutTimestamp'] }
+  | {
+      /** if specified, will be converted to a `timeoutTimestamp` in the future using {@link TimerService} */
```

Comment:
I read this once as something it will _do_ "in the future". Also the "if specified" is implied by documenting the optional value.
```suggestion
      /** converted using {@link TimerService} to a `timeoutTimestamp` value `timeoutRelativeSeconds` in the future */
```